scripts/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#put token in user's home dir
+printenv INDICO_API_TOKEN > ~/indico_api_token.txt
+#install additional test reqs and run smoke tests
+pip install requests-mock pytest
```

Comment:
what's the reasoning to not use the tests_requires with setup.py in the Dockerfile?

Reply:
I wasn't sure if that would stay around when setup.py test was deprecated